buildenv/jenkins/common/build.groovy

```diff
@@ -310,7 +310,15 @@ def build() {
     }
     stage('Java Version') {
         dir(OPENJDK_CLONE_DIR) {
-            sh "build/$RELEASE/images/$JDK_FOLDER/bin/java -Xjit -version"
+            // Include the leading ./ here so that jdkImageDir is suitable for
+            // use in find . -path '${jdkImageDir}'.
```

Comment:
Please explain this change. I don't see how this relates to use of the `find` utility.

Reply:
When we run `find . [..]`, the path to each file considered (other than `.` itself) will start with `./`. This is the path that we're checking with `-path`. So e.g. `find . -path build` will never match anything. Instead it to match the build directory it would be necessary to specify `-path ./build`.

Reply:
If the comment made reference to the use of `jdkImageDir` in the call to `archive_diagnostics()` it might be clearer.

This seems a little fragile: `archive_diagnostics()` will not accept any path. Maybe `archive_diagnostics()` should be adding the `./` prefix rather than rely on the caller to do so.

Reply:
I've removed the leading `./` here and the comment explaining it, and I've made `archive_diagnostics()` add it itself (when missing).